README.rst

```diff
@@ -2,278 +2,6 @@
-------------------------------
-
-Q: What happens if I don't move my content into a collection?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Content that doesn't end up in its own Collection will end up being automatically migrated to ``community.general`` during the devel freeze window.
-
-Q: Why would I want a dedicated collection?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-The benefits of claiming content are the following:
-
-* Source content is housed in a GitHub organization/repository of your choosing
-* Source content is subject to your own CI processes, decisions, and testing
-* Your own dedicated Issue and PR backlog
-* Ability to use more GitHub functionality, such as direct assignments, reviews, milestones and Project Boards
-
-Q: What will versioning and deprecation look like for Collections?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-* In ansible/ansible:
-
-  * There is a single version number which is over everything shipped in Ansible
-  * Doesn't use semver, uses X.Y (ie 2.9) as the major number
-  * Deprecations are done over 4 versions (~ 2 years)
-* In Collections
-
-  * Can be versioned and released independently to Ansible
-  * By convention, Galaxy requires a Collection to follow `semver (Semantic Versioning) <https://semver.org/>`_
-
-Details around versioning and deprecation policy are still being worked on, we will have a proposal up shortly
-
-
-Q: What should I do to move plugins across collections during migration?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-See `Migrating content to a different collection <https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#migrating-ansible-content-to-a-different-collection>`_ .
-
-
-Q: How can I fix bugs in Ansible 2.9?
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Please note that Ansible 2.9 no longer receives bugfixes. Only security issues can be fixed, and eventually it will be end of life with no more fixes accepted. See `ansible-core release cycle <https://docs.ansible.com/ansible/devel/reference_appendices/release_and_maintenance.html#ansible-core-release-cycle>`_ for whether 2.9 is still accepting security fixes or not.
-
-The process for fixing a security issue is as follows:
-
-1. PR for bug fix made against the Collection
-2. PR gets merged into Collection
-3. Raise PR directly against ``ansible/ansible:stable-2.9`` (ie not a backport) including a ``changelogs/fragments/`` file
-
-The changes in the PR against ``ansible/ansible:stable-2.9`` should be as close as possible to the changes in the collection original PR, and you should add a reference to the collection PR in the ``ansible/ansible`` PR.
+If you are here to discuss Ansible community collections or request a repository for a community collection under the ``ansible-collections`` GitHub organization, see the [communication guide](https://docs.ansible.com/ansible/devel/community/communication.html).
```

Comment:
```suggestion
If you are here to discuss Ansible community collections or request a repository for a community collection under the ``ansible-collections`` GitHub organization, see the `communication guide <https://docs.ansible.com/ansible/devel/community/communication.html>`_.
```

This looks weird at the moment:

```
see the [communication guide](https://docs.ansible.com/ansible/devel/community/communication.html)
```

I'm not 100% sure but I hope my suggestion will create a link.

Reply:
Yes, your suggestion is correct.

Reply:
oh, it's .rst, right, thanks! committed, PTAL